.github/workflows/docker.yml

```
@@ -0,0 +1,140 @@
+          platforms: ${{ matrix.platform }}
+          build-args: |
+            GIT_SHA=${{ github.sha }}
+#          push: ${{ github.event_name != 'pull_request' }}
+          cache-from: type=registry,ref=ghcr.io/ark-flow/arkflow:buildcache-${{ matrix.arch }}
+          cache-to: type=registry,ref=ghcr.io/ark-flow/arkflow:buildcache-${{ matrix.arch }}
+          # note that this is now the same as arroyo
+          target: arkflow
+          outputs: type=image,name=ghcr.io/ark-flow/arkflow,push-by-digest=true,name-canonical=true,push=true,store=true
+
+      - name: Export digest
+        run: |
```

actionlint (1.7.4)

property "build-single" is not defined in object type {build: {conclusion: string; outcome: string; outputs: {digest: string; imageid: string; metadata: string}}; meta: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}}

(expression)